### PR TITLE
EPMRPP-85839 || Add events for API KEYS

### PR DESCRIPTION
**Pull request by BlazarQSO**

### CODE REVIEW

File: app/src/components/main/analytics/events/ga4Events/profilePageEvent.js [hunk 1]
```diff
@@ -14,18 +14,57 @@
...
     type: lang,
     element_name: 'language_drop_down',
   }),
+
+  CLICK_API_KEYS_TAB_EVENT: {
+    ...BASIC_EVENT_PARAMETERS_PROFILE,
+    element_name: 'tab_api_keys',
+  },
+
+  CLICK_GENERATE_API_KEY_BUTTON: (isApiKey) => ({
```
Comment (Bam6ycha) [inline app/src/components/main/analytics/events/ga4Events/profilePageEvent.js:36]: Let's use camelCase for event creators.

File: app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.jsx [hunk 2]
```diff
@@ -56,13 +59,42 @@ const messages = defineMessages({
...
+  const onRevokeClick = (key) => {
+    dispatch(showModalAction({ id: 'revokeApiKeyModal', data: key }));
+    trackEvent(PROFILE_EVENTS.CLICK_REVOKE_BUTTON);
+  };
+
+  const onDocumentationClick = (event) => {
+    const { tagName } = event.target;
+
+    if (tagName === 'A') {
+      trackEvent(PROFILE_EVENTS.CLICK_DOCUMENTATION_LINK(true));
```
Comment (Bam6ycha) [inline app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.jsx:78]: Looks like this violates one of the clean code principles.
If we are using static bool arg, it is better to create two different functions

**Review by Bam6ycha — CHANGES_REQUESTED** [review]

File: app/src/components/main/analytics/events/ga4Events/profilePageEvent.js (hunk 1 above)
Comment (BlazarQSO) [reply app/src/components/main/analytics/events/ga4Events/profilePageEvent.js]: CONSTANT_CASE is used everywhere in event creators.

File: app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.jsx (hunk 2 above)
Comment (BlazarQSO) [reply app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.jsx]: Done.

**Review by Bam6ycha — APPROVED** [review]

**Review by AmsterGet — APPROVED** [review]

File: app/src/components/main/analytics/events/ga4Events/profilePageEvent.js (hunk 1 above)
Comment (AmsterGet) [reply app/src/components/main/analytics/events/ga4Events/profilePageEvent.js]: When migrating to GA4, we agreed to use CamelCase for event creators to better distinguish constant events from them.
So if there are some events with old naming that affected by particular task, let's provide new names for them according to above agreement.

**Pull request merged**